web/service_agreements_controller_test.go

```diff
@@ -21,22 +21,18 @@ func TestServiceAgreementsController_Create(t *testing.T) {
 	client := app.NewHTTPClient()
 	base := cltest.EasyJSONFromFixture("../internal/fixtures/web/hello_world_agreement.json")
 
+	account, err := app.Store.KeyStore.GetAccount()
+	assert.NoError(t, err)
+	base = base.Add("oracles", []string{account.Address.Hex()})
+
 	tests := []struct {
-		name      string
-		input     string
-		wantCode  int
-		id        string
-		signature string
+		name     string
+		input    string
+		wantCode int
 	}{
-		{
-			"basic",
-			base.String(),
-			200,
-			"0x8de92e4d6a2b527f1e3c39022ee0f1c177a6d874224fe54f5c4c0d5bcaa57d50",
-			"0xe86c1172d713c57b8df0d9f1ae91e4524cbb4e40719798ea48d3f777d2c893c000214b70376ddb661a8dc02fcadb911582f0e00e62eece65d3f9b8d0bfb6702201",
-		},
-		{"fails validation", base.Delete("payment").String(), 400, "", ""},
-		{"invalid JSON", "{", 400, "", ""},
+		{"success", base.String(), 200},
+		{"fails validation", base.Delete("payment").String(), 422},
+		{"invalid JSON", "{", 422},
```

Comment:
Can we add one case where we have `base` but with a lower case address asserting that it errors because of failed checksum violation? Whatever is easiest to cover that scenario (`base.Add("oracles", []string{strings.ToLower(account.Address.Hex())})`)

Reply:
Done